cli/tests/integrations/test_marathon_pod.py

```diff
@@ -129,6 +121,7 @@ def _assert_pod_show(pod_id):
 
     pod_json = json.loads(stdout.decode('utf-8'))
     assert pod_json['id'] == util.normalize_marathon_id_path(pod_id)
+    return pod_json
```

Comment:
This helper is a little confusing since it is named `_assert_x`, but also returns a value.